utils/uart_utils.c

```diff
@@ -81,11 +81,13 @@ char *uart_read(const int fd) {
 
   ssize_t rdlen = read(fd, buf, sizeof(buf) - 1);
   if (rdlen > 0) {
-    // printf("buf = %s\n", buf);
+#ifdef DEBUG
+    printf("buf = %s\n", buf);
```

Comment:
Is it necessary?

Reply:
I think all of macro flags during compiling should be reduce. But I will not solve in this pull request.

Reply:
Create a new issue for cleaning up existing codebase.

Reply:
New issue at #33